components/automate-ui/src/app/pages/team-management/team-management.component.ts

```diff
@@ -19,12 +27,18 @@ export class TeamManagementComponent implements OnInit {
   public sortedTeams$: Observable<Team[]>;
   public loading$: Observable<boolean>;
   public teamToDelete: Team;
-  public modalVisible = false;
+  public deleteModalVisible = false;
+  public createModalVisible = false;
+  public createTeamForm: FormGroup;
+  public creatingTeam = false;
+  public conflictErrorEvent = new EventEmitter<boolean>();
   public iamMajorVersion$: Observable<string>;
```

Comment:
We've got a type `IAMMajorVersion` in `entities/policy.model`. Let's use that, it's more specific than `string`.

Reply:
good catch, we're using string in some other places too